airbyte-webapp/src/config/defaultConfig.ts

```diff
@@ -5,9 +5,10 @@ const defaultConfig: Config = {
   healthCheckInterval: 20000,
   version: "dev",
   apiUrl: `${window.location.protocol}//${window.location.hostname}:8001/api`,
-  connectorBuilderApiUrl: `${window.location.protocol}//${window.location.hostname}:8080/`,
+  #connectorBuilderApiUrl: `${window.location.protocol}//${window.location.hostname}:8003/`, #FIXME: Uncomment this when enabling connector-builder
```

Comment:
Don't need to comment this out -- this is already there in master so feel free to just update the port here
```suggestion
  connectorBuilderApiUrl: `${window.location.protocol}//${window.location.hostname}:8003/`,
```